spec/services/users_service_spec.rb

```diff
@@ -28,6 +28,24 @@
         }
       )
     end
+
+    it 'creates an organization, user and membership' do
+      result = subject.register('email', 'password', 'organization_name')
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)